sys-boot/plymouth/plymouth-9999.ebuild

```diff
@@ -98,7 +108,7 @@ src_install() {
 	# looks like make install create /var/run/plymouth
 	# this is not needed for systemd, same should hold for openrc
 	# so remove
-	rm -rf "${D}"/var/run
+	rm -rf "${D}"/var/run || die
```

Comment:
`-f` nullifies the effect of `|| die`, and should therefore probably be removed.